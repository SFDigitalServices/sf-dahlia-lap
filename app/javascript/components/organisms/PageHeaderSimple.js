import React from 'react'

const PageHeaderSimple = ({ title }) => {
  return (
    <header className="lead-header short bg-vapor">
    	<div className="row full-width inner--3x">
    		<div className="large-12 columns ">
    			<hgroup className="lead-header_group">
    				<h1 className="lead-header_title small-serif c-oil">{title}</h1>
    			</hgroup>
    		</div>
    	</div>
    </header>
  )
}

export default PageHeaderSimple
