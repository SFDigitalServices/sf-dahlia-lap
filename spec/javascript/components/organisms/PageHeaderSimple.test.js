import React from 'react'
import renderer from 'react-test-renderer'
import PageHeaderSimple from 'components/organisms/PageHeaderSimple'
import sinon from 'sinon'

describe('PageHeaderSimple', () => {
  let title = "title1"

  test('should render succesfully', () => {
    const component = renderer.create(
      <PageHeaderSimple title={title} />,
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})
