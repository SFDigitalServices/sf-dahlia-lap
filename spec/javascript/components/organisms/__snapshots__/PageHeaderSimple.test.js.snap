// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeaderSimple should render succesfully 1`] = `
<header
  className="lead-header short bg-snow"
>
  <div
    className="row full-width inner--3x"
  >
    <div
      className="large-12 columns "
    >
      <hgroup
        className="lead-header_group"
      >
        <h1
          className="lead-header_title small-serif c-oil"
        >
          title1
        </h1>
      </hgroup>
    </div>
  </div>
</header>
`;
